conda/plugins/manager.py

```diff
@@ -249,6 +252,24 @@ def get_subcommands(self) -> dict[str, CondaSubcommand]:
             for subcommand in self.get_hook_results("subcommands")
         }
 
+    def get_virtual_packages(self) -> tuple[CondaVirtualPackage, ...]:
```

Comment:
Could we add a test for this?

Reply:
already did 😄 https://github.com/conda/conda/pull/13057/commits/279fedcaa546797af99a9dbd882c20289105beca

Reply:
Like codecov below shows that some branches aren't covered :D

Reply:
Ah i see what you mean